### PR TITLE
Humanize spell check language labels

### PR DESCRIPTION
**Pull request by t3chguy**
Like they are in the dropdown used to add languages

### CODE REVIEW

File: src/components/views/settings/SpellCheckSettings.tsx [hunk 1]
```diff
@@ -87,8 +90,9 @@ export default class SpellCheckLanguages extends React.Component<SpellCheckLangu
     };
 
     public render(): React.ReactNode {
+        const intl = new Intl.DisplayNames([getUserLanguage()], { type: "language", style: "short" });
```
Comment (richvdh) [inline src/components/views/settings/SpellCheckSettings.tsx:93]: does `style: "short"` actually do anything here, ooi?

If not, I'd favour omitting it.

File: src/components/views/settings/SpellCheckSettings.tsx [hunk 2]
```diff
@@ -18,10 +18,11 @@ import React from "react";
 
 import SpellCheckLanguagesDropdown from "../../../components/views/elements/SpellCheckLanguagesDropdown";
 import AccessibleButton, { ButtonEvent } from "../../../components/views/elements/AccessibleButton";
-import { _t } from "../../../languageHandler";
+import { _t, getUserLanguage } from "../../../languageHandler";
 
 interface ExistingSpellCheckLanguageIProps {
     language: string;
+    label?: string;
```
Comment (richvdh) [inline src/components/views/settings/SpellCheckSettings.tsx:25]: documentation, please. I know this isn't an exported class, but it still represents the public interface to `ExistingSpellCheckLanguage` and we should be in the habit of documenting all of our props.

Please include the behavior if `label` is omitted in the doc.

**Review by richvdh — APPROVED** [review]
lgtm modulo a couple of nits

File: src/components/views/settings/SpellCheckSettings.tsx (hunk 1 above)
Comment (t3chguy) [reply src/components/views/settings/SpellCheckSettings.tsx:93]: Yes.

image

The default is `long`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DisplayNames/DisplayNames#options

**Pull request merged**